cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -242,19 +252,21 @@ sensor_msgs::LaserScan scan_unifier_node::unifieLaserScans()
       ROS_DEBUG_STREAM("Converting scans to point clouds at index: " << i << ", at time: " << vec_laser_struct_.at(i).current_scan_msg.header.stamp << " now: " << ros::Time::now());
       try
       {
-        listener_.waitForTransform("/base_link", vec_laser_struct_.at(i).current_scan_msg.header.frame_id,
-            vec_laser_struct_.at(i).current_scan_msg.header.stamp, ros::Duration(3.0));
-
+        if (!listener_.waitForTransform(frame_, vec_laser_struct_.at(i).current_scan_msg.header.frame_id,
+            vec_laser_struct_.at(i).current_scan_msg.header.stamp, ros::Duration(3.0)))
+        {
+          continue;
```

Comment:
@ipa-mig @mig-em 
Well, here we silently skip a scan that cannot be transformed. Should we at least notify in some way?
e.g. `ROS_ERROR_STREAM("Not using scan during unification: " << vec_laser_struct_.at(i).current_scan_msg.header.frame_id,);`